docs/secure-connections/sauce-connect/setup-configuration/environment-variables.md

```diff
@@ -8,21 +8,133 @@ import useBaseUrl from '@docusaurus/useBaseUrl';
-| `http_proxy`<br/>`HTTP_PROXY`<br/>`all_proxy`<br/>`ALL_PROXY` | Sets an HTTP proxy to be used by Sauce Connect Proxy. It can be formatted as `http://hostname:port` or `hostname:port`.<br/><br/>On Linux and Mac environments, `http_proxy` and `https_proxy` variables can contain proxy credentials in the following format: `scheme://user:password@host:port` | Windows, Linux, Mac | [`--proxy`](/dev/cli/sauce-connect-proxy/#external-proxy-configuration) |
-| `no_proxy`<br/>`NO_PROXY` | Sets hostnames that will not be proxied, even when Sauce Connect Proxy is configured to use a proxy. Format as a comma-separated list. Subdomain wildcarding is supported when the hostname starts with a `.` Examples:<br/><br/>`saucelabs.com,spam.net`: Only requests hitting `saucelabs.com` and `spam.net` will not be proxied. All other requests will be proxied.<br/><br/>`.example.com`: All requests going to any subdomain of `example.com` will not be proxied (e.g., `one.example.com`, `asdf.example.com`) | Linux | n/a |
-| `SAUCE_ARES_FLAGS` | Sets advanced flags for DNS library use with Sauce Connect Proxy. For more information, see the [c-ares Documentation](http://c-ares.haxx.se/ares_init.html). | Windows, Linux, Mac | n/a |
 
-For best performance, use the [latest version of Sauce Connect Proxy](/secure-connections/sauce-connect/installation).
+## What You'll Need
+* See [Sauce Connect Proxy Basic Setup requirements](/secure-connections/sauce-connect/setup-configuration/basic-setup/#what-youll-need).
+
+
+## Running Sauce Connect Proxy with enviroment variables
+
+  <Tabs
+      defaultValue="Mac/Linux"
+      values={[
+        {label: 'Mac/Linux', value: 'Mac/Linux'},
+        {label: 'Windows', value: 'Windows'},
+      ]}>
+
+<TabItem value="Mac/Linux">
+
+Follow the steps below to configure Sauce Connect Proxy using enviroment variables in the current terminal.
+
+1. In your terminal window, set the following enviroment variables
+  ```bash
+  export SAUCE_USERNAME="your Sauce username"
+  export SAUCE_ACCESS_KEY="your Sauce access key"
+  ```
+2. Starting a new Sauce Connect Proxy will not require adding `--api-key` or `--user` keys.
+  ```bash
+  sc --region us-west
+  ```
+
+Alternatively, Sauce Connect Proxy enviroment variables may be added to one of the user environment configuration files, such as `~/.bash_profile`.
+
+1. To make enviroment variables always available, open `~/.bash_profile` in your prefered text editor.
+2. Add the variables, for example:
+ ```bash
+ export SAUCE_USERNAME="your Sauce username"
+ export SAUCE_ACCESS_KEY="your Sauce access key"
+ ```
+3. Start a new shell or a new terminal.
+4. Confirm that your environment variables have been set by typing `echo $SAUCE_USERNAME` in your terminal. The response should be your username value.
+5. Just as above, starting a new Sauce Connect Proxy will not require adding `--api-key` or `--user` keys.
+ ```bash
+ sc --region us-west
+ ```
```

Comment:
Curious...I know you can technically also set `region` as an env var. If I did that, would I start a new tunnel by simply typing `sc` at the CLI?

Reply:
yup